.github/workflows/gemini-readability.yml

```diff
@@ -66,6 +66,35 @@ jobs:
           echo "::debug::Wrote configuration file to ${READABILITY_MD}"
           echo "output-file=${READABILITY_MD}" >> "${GITHUB_OUTPUT}"
 
+      - name: 'Fetch Safe Discussion History'
+        id: 'fetch_history'
+        env:
+          GH_TOKEN: '${{ github.token }}'
+          PR_NUMBER: '${{ github.event.pull_request.number }}'
+          REPOSITORY: '${{ github.repository }}'
+        run: |
+          # Fetch review comments.
+          # We grab the Short Commit SHA so the LLM knows if the comment is "Fresh" or "Stale".
+          DISCUSSION=$(gh api \
+            --paginate \
+            "repos/${REPOSITORY}/pulls/${PR_NUMBER}/comments" \
+            --jq '
+              .[]
+              | select(
+                  .user.login == "github-actions[bot]" or .user.login == "github-actions"
+                )
+              | "[Commit: " + (.commit_id | .[0:7]) + "] [File: " + .path + " Line: " + (if .line then (.line|tostring) else (.original_line|tostring) end) + "] **" + .user.login + "**: " + .body' \
```

Comment:
This is complex enough that I think you should explain it in the comment at the top. I think I understand but an example would make this more maintainable in a year when some new dev is looking at it.